sdk/appconfiguration/azure-appconfiguration/azure/appconfiguration/_azure_appconfiguration_client.py

```diff
@@ -153,8 +153,8 @@ def list_configuration_settings(
         :param label_filter: filter results based on their label. '*' can be
          used as wildcard in the beginning or end of the filter
         :type label_filter: str
-        :keyword datetime accept_datetime: retrieve ConfigurationSetting existed at this datetime
-        :keyword List[str] fields: specify which fields to include in the results. Leave None to include all fields
+        :keyword str accept_datetime: retrieve ConfigurationSetting existed at this datetime
```

Comment:
Why this is a string? from the sample code below, seems like this is a datetime object.

Reply:
It is `str`in generated operation: https://github.com/Azure/azure-sdk-for-python/blob/aced1e299c25878d9b96325690564bf6edf89c33/sdk/appconfiguration/azure-appconfiguration/azure/appconfiguration/_generated/operations/_azure_app_configuration_operations.py#L42

We need to update the sample.